app/code/Magento/Contact/etc/webapi.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
+<routes xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Webapi:etc/webapi.xsd">
+    <route url="/V1/contact" method="POST">
+        <service class="Magento\Contact\Api\ContactInterface" method="send" />
+        <resources>
+            <resource ref="anonymous" />
+        </resources>
+    </route>
+</routes>
```

Comment:
Maybe we also can introduce the GraphQL entry point for these API. It easy to implement but it guarantees that functionality will be used in new approaches (PWA)